src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,35 @@
+package calculator;
+
+public class StringCalculator {
+
+    public int add(String text) {
+        Validation valid = new Validation();
+        Operation operation = new Operation();
+
+        //선처리1: 입력이 없을 경우는 0
+        if(valid.isNull(text)){
+            return 0;
+        }
+
+        //선처리2: 입력 문자열이 하나의 숫자인 경우 그대로 리턴.
+        if(valid.isInt(text)){
+            int sum = Integer.parseInt(text);
+            if(sum < 0){
+                throw new RuntimeException("입력문자가 음수입니다.");
+            }
+            return sum;
+        }
+
+        // 추가 구분자 확인. 구분자 존재시 문장에서 구분자 문장제거
+        String removeOperText = operation.sepOperatorFromText(text);
+        String operator = operation.getOperator();
+
+        //가지고있는 구분자로 숫자 나누기
+        String[] tokens= removeOperText.split(operator);
```

Comment:
```suggestion
        String operator = operation.getOperator();

        //가지고있는 구분자로 숫자 나누기
        String[] tokens= removeOperText.split(operator);
```

변수 앞을 띄우거나 띄우지 않거나, 작성된 코드의 스페이스 규칙이 일관성이 없네요. IDE의 [Reformat code﻿](https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html)를 수행하는 부분을 습관화해보시면 좋겠습니다.

Reply:
습관 들이겠습니다! :)